contrib/exporters/core/extend.go

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package core
+
+import (
+	"strings"
+	"encoding/json"
+	"github.com/spf13/viper"
+	"github.com/skydive-project/skydive/logging"
+	"github.com/skydive-project/skydive/api/client"
+)
+
+// for no transform extensions
+type extendNone struct {
+}
+
+func NewExtendNone(cfg *viper.Viper) (interface{}, error) {
+	logging.GetLogger().Infof("entering NewExtendNone")
+        return &extendNone{}, nil
+}
+
+func (e *extendNone)  Extend(in interface{}) interface{} {
+	return in
```

Comment:
did you run the code through `gofmt -s -w` ? asking as a notice two-spaces before `Extend`